tests/app/dao/test_ft_billing_dao.py

```diff
@@ -59,19 +59,20 @@ def set_up_yearly_data():
 
 
 def test_fetch_billing_data_for_today_includes_data_with_the_right_status(notify_db_session):
+    today = convert_utc_to_bst(datetime.utcnow())
     service = create_service()
     template = create_template(service=service, template_type="email")
+
     for status in ['created', 'technical-failure']:
         create_notification(template=template, status=status)
-
-    today = convert_utc_to_bst(datetime.utcnow())
     results = fetch_billing_data_for_day(today.date())
     assert results == []
-    for status in ['delivered', 'sending', 'temporary-failure']:
+
+    for status in ['delivered', 'sending', 'temporary-failure', 'pending']:
```

Comment:
This check seems to be testing the same behaviour (or at least less of it) than `test_fetch_billing_data_for_today_includes_data_with_the_right_key_type`? If so, we could probably remove this section from this test altogether?

Reply:
In fact, is this whole test adding any more value over test_fetch_billing_data_for_today_includes_data_with_the_right_key_type? The negative cases for statuses that are not billable are tested implicitly there too

Reply:
i think you mean `test_fetch_billing_data_for_day_bills_correctly_for_status` here? in which case, i agree and i think this test can be deleted.

though that said, i have some misgivings around both of these tests in general as if it fails it's not apparent _which_ notification status types is behaving unexpectedly

Reply:
I did, thanks.

And yes. Not only that, if you made a mistake to include one status you didn't mean to, whilst also excluding a different status you didn't mean to, the test would pass as it just tests the number. I'll have a think if there is an easy way to test this more thoroughly

Reply:
@leohemsted so I've removed this test. I couldn't think of a way to easily improve the other one though. Open to suggestions but if not then I think this may be ready to go.